docs/cpp/cmake-linux.md

```diff
@@ -0,0 +1,176 @@
+
+![Project contents](images/cpp/cmake-project-contents.png)
+
+### Select a kit
+
+Before you can use the CMake tools extension to build a project, you need to configure it to know about the compilers on your system. Do that by scanning for what CMake calls 'kits'. A kit represents a toolchain, which is the compilers, linkers, and other tools used to build your project. To scan for kits:
+
+1. Open the Command Palette (`kb(workbench.action.showCommands)`) and run **CMake: Select a Kit**. The extension will automatically scan for kits on your computer and create a list of compilers found on your system.
+
+1. Select the compiler you want to use. For example, depending on the compilers you have installed, you might see something like:
+
+    ![Select the kit](images/cpp/cmake-selectkit.png)
+
+## Configure Hello World
+
+There are two things you must do to configure your CMake project: select a kit (which you just did) and select a variant.  
+
+The kit you selected previously is shown in the status bar. For example:
+
+![Selected kit in status bar](images/cpp/cmake-kit-statusbar.png)
+
+To change the kit, you can click on the kit in the status bar, or run the **CMake: Select a kit** again from the Command Palette. If you don't see the compiler you're looking for, you can edit the `cmake-tools-kits.json` file in your project and add it manually.
+
+### Select a variant
+
+A variant contains instructions for how to build your project. By default, the CMake Tools extension provides four variants, each corresponding to a default build type: `Debug`, `Release`, `MinRelSize`, and `RelWithDebInfo`. These options provide the following:
+
+`Debug`: disables optimizations and includes debug info.
+`Release` : Includes optimizations but no debug info.
+`MinRelSize` : Optimizes for size. No debug info.
+`RelWithDebInfo` : Optimizes for speed and includes debug info.
+
+To select a variant, open the Command Palette (`kb(workbench.action.showCommands)`) run the **CMake: Select Variant** command.
+
+![Select variant](images/cpp/cmake-select-variant.png)
+
+Select **Debug** to include debug information with your build.  
+
+![Select debug variant type](images/cpp/cmake-variant-type.png)
+
+The selected variant will appear in the status bar next to the active kit.  
+
+### CMake: Configure
+
+Now that you've selected a kit and a variant, open the Command Palette (`kb(workbench.action.showCommands)`) and run the **CMake: Configure** command to configure your project. This generates build files in the project's build folder using the kit and variant you selected.
+
+## Build hello world
+
+After configuring your project, you're ready to build. Open the Command Palette (`kb(workbench.action.showCommands)`) and run the **CMake: Build**, or simply select the **Build** button from the status bar.  
```

Comment:
missing the word "command" after the command.

Reply:
fixed that one. Plus another that was like it.